src/app/not-found.js

```diff
@@ -0,0 +1,28 @@
+import Image from 'next/image';
+import notFound from '@public/not-found.webp';
+import Link from 'next/link';
+
+export default async function NotFound() {
+  return (
+    <main className="p-2 pt-20">
+      <div className="flex h-full flex-col items-center justify-center gap-12 text-hdark lg:px-24">
+        <div className="relative">
+          <Image src={notFound} alt="404" width={900} height={743} />
```

Comment:
I would rather prefer a smaller picture

width={400} height={400} for example :)